scopes/defender/ui/test-compare/compare-tests.module.scss

```diff
@@ -0,0 +1,44 @@
+.checkboxContainer {
```

Comment:
seems like this is repeating, try to find a way to extract a template from it 
```
<CompareLayoutPreset topLeft={} topRight={} bottomLeft={} bottomRight={}>
<CompareLayoutPreset baseLeft={} baseRight={} compareLeft={} compareRight={}>
```

also you can make a couple of presets, like
```
<Compare4BlockPreset>
<Compare2BlockPreset>
```

CutOutBoard? Preset? Layout? Template